Core.ApplicationServices/System/ItSystemService.cs

```diff
@@ -320,6 +323,17 @@ public bool CanChangeNameTo(int organizationId, int systemId, string newName)
             return ValidateNameChange(organizationId, systemId, newName).IsNone;
         }
 
+        public Result<ResourcePermissionsResult, OperationError> GetPermissions(Guid uuid)
+        {
+            return GetSystem(uuid).Transform(result => ResourcePermissionsResult.FromResolutionResult(result, _authorizationContext));
+        }
+
+        public Result<ResourceCollectionPermissionsResult, OperationError> GetCollectionPermissions(Guid organizationUuid)
+        {
+            return _organizationService.GetOrganization(organizationUuid)
+                .Bind<ResourceCollectionPermissionsResult>(organization => ResourceCollectionPermissionsResult.FromOrganizationId<ItSystem>(organization.Id, _authorizationContext));
```

Comment:
this one does not have an "error" return, so it should just be a `Select`. Remember the difference between the two methods

It can be rewritten to (notice there is no need to specify the type since there is only one path here)
```
return _organizationService
                .GetOrganization(organizationUuid)
                .Select(organization => ResourceCollectionPermissionsResult.FromOrganizationId<ItSystem>(organization.Id, _authorizationContext));
```